website/content/docs/security/acl/acl-policies.mdx

```diff
@@ -393,115 +395,141 @@ New installations of Consul ship with the following built-in policies.
 
 The `global-management` policy grants unrestricted privileges to any token linked to it. The policy is assigned the reserved ID of `00000000-0000-0000-0000-000000000001`. You can rename the global management policy, but Consul will prevent you from modifying any other attributes, including the rule set and datacenter scope.
 
-### Namespace Management <EnterpriseAlert inline />
-
-The `namespace-management` policy will be injected into all namespaces you create. The policy will be assigned a randomized UUID and can be managed as a normal, user-defined policy within the namespace. This feature was added in Consul Enterprise 1.7.0.
+The following are the policy rules for the `global-management` policy.
```

Comment:
```suggestion
The following policy rules define the `global-management` policy.
```

Reply:
FWIW, I'm not sure how useful it is to show the policy rules for the built-in global-management and namespace-management policies. I don't think users will need to reference policy rules for built-in policies that often, and the policies are long and clutter these sections. 

I included them because I tracked down the rules, and for completeness, and because I like the docs having many examples of working policies to reference.

I thought making the policy rules collapsable/expandable sections would be a good compromise, but wasn't sure how to do a collapsable section.